lib/osf-components/addon/components/open-resource/styles.scss

```diff
@@ -0,0 +1,16 @@
+.OutputResource_figure {
+    display: flex;
+    font-weight: 100;
+    padding-left: 9px;
+    padding-top: 3px;
+}
+
+img {
+    padding-right: 3px;
+    height: 22px;
+    width: 22.5px;
+}
+
+figcaption {
+    padding-left: 5px;
+}
```

Comment:
These css rules will apply to all `<img>` and all `<figcaption>` in the app, so these should be rewritten as local-classes or be nested rules within a local-class

Reply:
Ok just to confirm even though it is the component's SCSS file it still needs to have its own local class?

Reply:
This is now updated, marking as resolved.

Reply:
Yea, it will impact other tags outside of the component (speaking from experience 😅 ). Another gotcha is styling `:global(.class)` without a qualifying local-class too